### PR TITLE
Create Video Hub App project

### PR DESCRIPTION
**Pull request by whyboris**
*Video Hub App* is an open source software built with Electron and Angular that compiles to Windows, Mac, and Linux applications.

It is actively developed and is used by hundreds of people.

Thank you for your consideration 🙇

### CODE REVIEW

File: _data/projects/video-hub-app.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: The fastest way to browse and search for videos on your computer
+site: https://github.com/whyboris/Video-Hub-App
+tags:
+- oss
+- video
+- angular
+- electron
+- javascript
+upforgrabs:
+  name: Video Hub App
```
Comment (ritwik12) [inline _data/projects/video-hub-app.yml]: @whyboris This should be "good first issue" as per the label link that you have mentioned below.

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/video-hub-app.yml (hunk 1 above)
Comment (whyboris) [reply _data/projects/video-hub-app.yml]: Sorry misunderstood 😅 will fix

**Comment by whyboris** [comment]
Thank you @ritwik12 for catching that! Fixed 😄

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@whyboris Thanks!!!

**Pull request merged**